src/lib/components/csvExportBox.svelte

```diff
@@ -0,0 +1,367 @@
+            }
+        }
+
+        if (!bucketName && bucketId) {
+            try {
+                const bucket = await sdk
+                    .forProject(page.params.region, page.params.project)
+                    .storage.getBucket({ bucketId });
+                bucketName = bucket.name;
+            } catch (error) {
+                console.error('Failed to fetch bucket name:', error);
+                bucketName = null;
+            }
+        }
+
+        if (tableId && tableName === null) {
+            exportItems.delete(exportData.$id);
+            return;
+        }
+
+        const existing = exportItems.get(exportData.$id);
+
+        const isDone = (s: string) => s === 'completed' || s === 'failed';
+        const isInProgress = (s: string) => ['pending', 'processing'].includes(s);
+
+        const shouldSkip =
+            (existing && isDone(existing.status) && isInProgress(status)) ||
+            existing?.status === status;
+
+        if (shouldSkip) return;
+
+        exportItems.set(exportData.$id, {
+            status,
+            table: tableName ?? current?.table,
+            bucketId: bucketId,
+            bucketName: bucketName,
+            fileName: fileName,
+            errors: exportData.errors || []
+        });
+        exportItems = new Map(exportItems);
+
+        if (status === 'completed' || status === 'failed') {
+            await showCompletionNotification(tableName ?? tableId, bucketId, fileName, exportData);
+        }
+    }
+
+    function clear() {
+        exportItems = new Map();
+    }
+
+    function graphSize(status: string): number {
+        switch (status) {
+            case 'pending':
+                return 10;
+            case 'processing':
+                return 60;
+            case 'completed':
+            case 'failed':
+                return 100;
+            default:
+                return 30;
+        }
+    }
+
+    function text(status: string, tableName = '', bucketName = '') {
+        const table = tableName ? `<b>${tableName}</b>` : '';
+        const bucket = bucketName ? `<b>${bucketName}</b>` : 'bucket';
+        switch (status) {
+            case 'completed':
+            case 'failed':
+                return `Export to ${bucket} ${status}`;
+            case 'processing':
+                return `Exporting ${table} to ${bucket}`;
+            default:
+                return 'Preparing export...';
+        }
+    }
+
+    onMount(() => {
+        sdk.forProject(page.params.region, page.params.project)
+            .migrations.list({
+                queries: [
+                    Query.equal('destination', 'CSV'),
+                    Query.equal('status', ['pending', 'processing'])
+                ]
+            })
+            .then((migrations) => {
+                migrations.migrations.forEach(updateOrAddItem);
+            });
+
+        return sdk.forConsoleIn(page.params.region).client.subscribe('console', (response) => {
```

Comment:
`client.subscribe` is deprecated for realtime, please use `realtime.forConsole(region, channel, callback)`.